### DISCUSSION TITLE
tracking from amd.com

### DISCUSSION THREAD

**Discussion by untyped1** [jef/streetmerchant#970] [Help]
Is there a solution where we can make streetmerchant restart when it gets stuck because a website is denying access? 
On AMD.com after 10mins I get Status Code Error 403. The same will happen if I do a manual refresh many times or set auto refresh using a chrome extension.
I noticed if I terminate the job and run it again it will work fine. So a solution could be just to reset the job if it gets stuck on such an issue?

**Comment by ketz** [comment]
I'm having the same issue. Sometimes it gets stuck for me instantly. It does 3 pings on the AMD site, returns 3 403's and then the script stays there doing nothing until I stop it
